.github/workflows/validate.yml

```diff
@@ -0,0 +1,81 @@
+# Runs cargo check, test, fmt and clippy on every push for default members.
+# Based on https://github.com/actions-rs/meta/blob/master/recipes/quickstart.md
+
+on: [push]
+
+name: Validate
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install native dependencies
+        run: sudo apt-get update && sudo apt-get install libasound2-dev
+
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
+
+      - name: Run cargo check
+        uses: actions-rs/cargo@v1
+        with:
+          command: check
+
```

Comment:
[Cargo check is a subset of clippy](https://stackoverflow.com/questions/57449356/is-cargo-clippy-a-superset-of-cargo-check), so we don't need this job.
```suggestion
```

Reply:
Aha, Nice!